lib/ansible/modules/remote_management/manageiq/manageiq_provider.py

```diff
@@ -181,6 +181,19 @@
         required: false
         default: null
         description: The CA bundle string with custom certificates. defaults to None.
+
+  auth_type:
```

Comment:
is this indented correctly ?
it looks wrong ... but it may be the github display ( didn't pull and checked ... )

Reply:
Its fine.